src/plugins/workspace/server/integration_tests/routes.test.ts

```diff
@@ -577,15 +577,21 @@ describe('workspace service api integration test when savedObjects.permission.en
         .post(root, `/api/workspaces`)
         .send({
           attributes: omitId(testWorkspace),
-          permissions: { invalid_type: { users: ['foo'] } },
+          settings: {
+            permissions: { invalid_type: { users: ['foo'] } },
+            dataSources: [],
```

Comment:
Nit: Though workspace is an experimental feature, I still not recommend we do the breaking change in the API side. Maybe next time we can add a parameter.